pkg/intel-gmmlib

```diff
@@ -0,0 +1,57 @@
+[mirrors]
+https://github.com/intel/gmmlib/archive/refs/tags/intel-gmmlib-21.1.1.tar.gz
+
+[vars]
+filesize=737024
+sha512=06becc986eb04c5243389686121bcc597ab16620fcb47943de14f901938dd906ad884b132da0e46ff939e7e599899d4262006efe19d14b3ed87b6f799ca40787
+desc='Intel Graphics Memory Management Library'
+pkgver=1
+tardir=gmmlib-intel-gmmlib-21.1.1
+
+[deps]
+libc
+
+[deps.host]
+cmake
+
+[build]
+# Build fails on x86 if stack protector is on
+# Nobody needs this on 32-bit but at least it compiles
+case "$A" in i[3-6]86) \
+ sed -i 's/-fstack-protector/-fno-stack-protector/g' \
```

Comment:
i think we can just unconditionally remove stack-protector, as in `s/-fstack-protector//g` for all archs, as sabotage's convention is that all CFLAGS should be user-, not package-controlled.

Reply:
Fixed and force pushed!